cmd/algorand-indexer/daemon.go

```diff
@@ -231,6 +234,19 @@ var daemonCmd = &cobra.Command{
 				genesisReader := importer.GetGenesisFile(genesisJSONPath, bot.Algod(), logger)
 				_, err := importer.EnsureInitialImport(db, genesisReader, logger)
 				maybeFail(err, "importer.EnsureInitialImport() error")
+
+				// sync local ledger
+				nextDBRound, err := db.GetNextRoundToAccount()
+				maybeFail(err, "Error getting DB round")
+				if nextDBRound > 0 {
+					if catchpoint != "" {
```

Comment:
we have a check for this in processor, https://github.com/algorand/indexer/blob/d057e1c0ab061f01fc9549c97685327938686cbd/processor/blockprocessor/block_processor.go#L52-L54. maybe also add one in the migration method?

Reply:
This is the opposite check, we need to avoid catching up if the catchpoint if ahead of the desired round (i.e. if you're starting a new node and provide a catchpoint, we shouldn't initialize anything)